services/shhext/api.go

```diff
@@ -488,29 +488,19 @@ func (api *PublicAPI) SendGroupMessage(ctx context.Context, msg chat.SendGroupMe
 }
 
 func (api *PublicAPI) processPFSMessage(msg *whisper.Message) error {
-	var privateKey *ecdsa.PrivateKey
-	var publicKey *ecdsa.PublicKey
-
-	// Msg.Dst is empty is a public message, nothing to do
-	if msg.Dst != nil {
-		// There's probably a better way to do this
-		keyBytes, err := hexutil.Bytes(msg.Dst).MarshalText()
-		if err != nil {
-			return err
-		}
 
-		privateKey, err = api.service.w.GetPrivateKey(string(keyBytes))
-		if err != nil {
-			return err
-		}
+	privateKeyID := api.service.w.SelectedKeyPairID()
+	if privateKeyID == "" {
+		return errors.New("No key selected")
```

Comment:
```suggestion
		return errors.New("no key selected")
```

Errors should be lowercase.